app/code/Magento/Swatches/view/frontend/web/js/swatch-renderer.js

```diff
@@ -919,11 +919,7 @@ define([
                 }
             );
 
-            if (typeof result != 'undefined' && result.oldPrice.amount !== result.finalPrice.amount) {
-                $(this.options.slyOldPriceSelector).show();
-            } else {
-                $(this.options.slyOldPriceSelector).hide();
-            }
+            $product.find(this.options.slyOldPriceSelector)[(typeof result != 'undefined' && result.oldPrice.amount !== result.finalPrice.amount) ? 'show' : 'hide']();
```

Comment:
Teh line is too long now, I would suggest a variable like

```
isShow = typeof result != 'undefined' && result.oldPrice.amount !== result.finalPrice.amount;
```

Reply:
Applied changes and pushed again.

Reply:
@sarfarazbheda please do not change unneded lines and remove brackets around `(isShow)`.

After that squash changes into a single commit.

Reply:
yes, Made changes and pushed branch again

Reply:
@sarfarazbheda I still see unneeded lines of code changed. No lines besides 922-926 should be changed, like it was before.

Reply:
Reverted my old changes and committed code